### PR TITLE
automatic focusing of the search bar when visiting the /search page

### PR DESCRIPTION
**Pull request by M2K3K5**

### CODE REVIEW

**Comment by RiadhAdrani** [comment]
The funny part is that I implemented it in my personal website and forgot to apply it in the template 🤷‍♂️

**Comment by M2K3K5** [comment]
hhahah lol, then take a look at what else you've missed :D
I've been using your template for a few days now and I think it's just great. I think I'll add a few things too

**Comment by M2K3K5** [comment]
Hey, I just visited your website on my cell phone and noticed that the search bar is focused on every page. Because of the cell phone, the cell phone keyboard opens every time I visit the site. Would you like to double check my pull request and apply my idea to focus the search bar only on the /search path (if the user really wants to search something)

**Comment by RiadhAdrani** [comment]
Hello,
I will be reviewing your prs soon, (mostly tomorrow), sorry for the delayed response !

File: src/lib/components/SearchPage.svelte [hunk 1]
```diff
@@ -28,6 +29,13 @@
 			const state = window.history.state;
 
 			window.history.replaceState(state, '', url);
+			
+			const path = window.location.pathname;
+			if (path.startsWith('/search')) {
```
Comment (RiadhAdrani) [inline src/lib/components/SearchPage.svelte]: Nice, but you didn't take into consideration the base, import these at the top 

```ts
import { page } from '$app/stores';
import { base } from '$app/paths';
```

and replace this line with :
```ts
if ($page.url.pathname.startsWith(`${base}/search`)) {
```

and we should be good to go !

**Review by RiadhAdrani — CHANGES_REQUESTED** [review]

**Review by RiadhAdrani — APPROVED** [review]
LGTM

**Pull request merged**